lib/core/src/sync/model/data.rs

```diff
@@ -98,6 +99,7 @@ pub(crate) struct SendSyncData {
     pub(crate) refund_private_key: String,
     pub(crate) payer_amount_sat: u64,
     pub(crate) receiver_amount_sat: u64,
+    pub(crate) timeout_block_height: u64,
```

Comment:
Do we need to set a serde default now incase we are syncing older data?

Reply:
Sounds like we got the first schema version update! I'll set it as minor and add the serde::default as you mentioned.